changelog/v1.10.0-release-note.md

```diff
@@ -0,0 +1,106 @@
+# v1.10.0 Release Note
+
+## Summary
+**v1.10.0 has a hardfork, which will be activated at block height
+22,991,401 (ETA is around 04/04/2023 11pm UTC) on IoTeX mainnet.**
+
+All nodes **must** upgrade to this release, or otherwise the node won't be able
+to sync with the IoTeX blockchain after the activation block.
+
+Also we'll use this release opportunity to disable the auto-upgrader, since the
+auto-upgrader is a fixed script that cannot adapt perfectly well to each release
+scenario, and has caused a couple of failed upgrades in the past.
+
+In the past, if you use the one-line upgrader tool to run your IoTeX node, please
+continue to use it to do the v1.10.0 upgrade. This time the script will turn off
+the auto-upgrader (if it was turned on before). Run the script below just as before.
+```bash
+sudo bash # If your docker requires root privilege
+bash <(curl -s https://raw.githubusercontent.com/iotexproject/iotex-bootstrap/master/scripts/setup_fullnode.sh)
+```
+
+If you use the very long `docker run` command (like below) to run your IoTeX node,
+don't worry about the auto-upgrader since it is **not** turned on. Continue to 
+upgrade your node using the v1.10.0 image:
+```
+docker stop iotex
+docker rm iotex
+docker run -d --restart on-failure --name iotex \
+        -p 4689:4689 \
+        -p 8080:8080 \
+        -v=$IOTEX_HOME/data:/var/data:rw \
+        -v=$IOTEX_HOME/log:/var/log:rw \
+        -v=$IOTEX_HOME/etc/config.yaml:/etc/iotex/config_override.yaml:ro \
+        -v=$IOTEX_HOME/etc/genesis.yaml:/etc/iotex/genesis.yaml:ro \
+        iotex/iotex-core:v1.10.0 \
+        iotex-server \
+        -config-path=/etc/iotex/config_override.yaml \
+        -genesis-path=/etc/iotex/genesis.yaml
+```
```

Comment:
tested with xianhua, if user run the node using `docker run`, the one-line upgrader will also work, but it requires to install `docker-compose`, if not installed earlier. This may cause inconvenience/usability issue for user.

auto-upgrade is an option only in one-line upgrader, so it is easier to have the instruction like this now:
1. if you use one-line upgrader, continue to use it
2. if you user `docker run`, continue to use it (auto-upgrade is NOT an issue in this case)